mim/_internal/export/common.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+# initialization template for __init__.py
+_init_str = """
+import os
+import os.path as osp
+
+all_files = os.listdir(osp.dirname(__file__))
+
+for file in all_files:
+    if (file.endswith('.py') and file != '__init__.py') or '.' not in file:
+        exec(f'from .{osp.splitext(file)[0]} import *')
+"""
+
+# import pack path for tools
+_import_pack_str = """
+import os
+import sys
+sys.path.append(os.path.dirname(__file__)+'/../')
+import pack
+
+"""
+
+BUILDER_TRANS = {
```

Comment:
What is the meaning of `TRANS`?

Reply:
It is a dict used to detect whether patches need to be used.
```
    def _match_alias_registry(self, node) -> Optional[str]:
        match_patch_key = None
        for key, list_value in BUILDER_TRANS.items():
            for alias in node.names:
                if alias.name in list_value:
                    match_patch_key = key
                    break

            if match_patch_key is not None:
                break
        return match_patch_key

    def visit_ImportFrom(self, node):
        matched = self._match_alias_registry(node)
        if matched is not None:
            # In an ideal scenario, the `ImportResolverTransformer` would
            # modify the import sources of all `Registry` from downstream
            # algorithm libraries (`mmdet`) to `pack`, for example, convert
            # `from mmdet.models import DETECTORS` to
            # `from pack.models import DETECTORS`.

            # However, some algorithm libraries, such as `mmpose`, provide
            # aliases for `MODELS`, `TASK_UTILS`, and other registries,
            # as seen here: https://github.com/open-mmlab/mmpose/blob/537bd8e543ab463fb55120d5caaa1ae22d6aaf06/mmpose/models/builder.py#L13.  # noqa: E501

            # For these registries with aliases, we cannot directly import from
            # `pack.registry` because `pack.registry` is copied from
            # `mmpose.registry` and does not contain these aliases.

            # Therefore, we gather all registries with aliases under
            # `mim.utils.mmpack.patch_utils` and hardcode the redirection
            # of import sources.
            if matched == 'MODELS':
                node.module = 'mim.utils.mmpack.patch_utils.patch_model'
            elif matched == 'TASK_UTILS':
                node.module = 'mim.utils.mmpack.patch_utils.patch_task'
            node.level = 0
            return node
    ...

```
Ok, I found it conflicting with a keyword. **I need to rename it.**
![image](https://github.com/open-mmlab/mim/assets/69315544/7ab5ec42-1dfc-4312-8275-65da0e8ec442)

Reply:
What kind of name should I use？

Reply:
`OBJECTS_TO_BE_PATCHED`